.eslintrc.js

```diff
@@ -202,54 +202,69 @@ module.exports = {
     },
     {
       files: ["site/**"],
+      parserOptions: {
+        tsconfigRootDir: `${__dirname}/site`,
+        project: "tsconfig.json",
+      },
       rules: {
         "no-restricted-imports": [
           "error",
           {
-            name: "@mui/material",
-            importNames: ["Link"],
-            message:
-              "Please use the custom src/components/Link component instead to ensure Next.js and MUI compatibility.",
-          },
-          {
-            name: "@mui/material/Link",
-            message:
-              "Please use the custom src/components/Link component instead to ensure Next.js and MUI compatibility.",
-          },
-          {
-            name: "next",
-            importNames: ["Link"],
-            message:
-              "Please use the custom src/components/Link component instead to ensure Next.js and MUI compatibility.",
-          },
-          {
-            name: "next/link",
-            message:
-              "Please use the custom src/components/Link component instead to ensure Next.js and MUI compatibility.",
-          },
-          {
-            name: "@mui/material",
-            importNames: ["Button", "TextField", "Popover"],
-            message:
-              "Please use the custom wrapper component in src/component instead.",
-          },
-          {
-            name: "@mui/material/Button",
-            importNames: ["default"],
-            message:
-              "Please use the custom src/components/Button component instead.",
-          },
-          {
-            name: "@mui/material/TextField",
-            importNames: ["default"],
-            message:
-              "Please use the custom src/components/TextField component instead.",
-          },
-          {
-            name: "@mui/material/Popover",
-            importNames: ["default"],
-            message:
-              "Please use the custom src/components/Popover component instead.",
+            paths: [
```

Comment:
Because I needed to use `patterns` for `fs-extra`, I had to introduce `path`, which created a whitespace diff here. Rule details: https://eslint.org/docs/rules/no-restricted-imports. Existing rules should still work – I tested them locally.